src/app/downloads/downloads.component.ts

```diff
@@ -2,26 +2,26 @@ import { Component, OnInit, OnDestroy } from '@angular/core';
 import { Router } from '@angular/router';
 import { Store } from '@ngrx/store';
 import { Subscription } from 'rxjs/Subscription';
+import { Observable } from 'rxjs/Observable';
 import { CastleService } from '../core';
-import { CastlePodcastMetricsAction, CastleEpisodeMetricsAction,
-  GoogleAnalyticsEventAction, CastleEpisodeChartToggleAction } from '../ngrx/actions';
+import * as ACTIONS from '../ngrx/actions';
 import { RouterModel, EpisodeModel, PodcastModel, ChartType, MetricsType,
   CHARTTYPE_PODCAST, INTERVAL_DAILY, EPISODE_PAGE_SIZE, METRICSTYPE_DOWNLOADS, getMetricsProperty } from '../ngrx';
-import { selectRouter, selectEpisodes, selectPodcasts } from '../ngrx/reducers';
+import { selectRouter, selectEpisodes, selectPodcasts, selectLoading, selectLoaded, selectErrors } from '../ngrx/reducers';
 import { filterPodcastEpisodePage } from '../shared/util/metrics.util';
 import * as dateUtil from '../shared/util/date';
 import { isPodcastChanged, isBeginDateChanged, isEndDateChanged, isIntervalChanged } from '../shared/util/filter.util';
 
 @Component({
   template: `
-    <prx-spinner *ngIf="isPodcastLoading || isEpisodeLoading" overlay="true"
+    <prx-spinner *ngIf="loading$ | async" overlay="true"
                  loadingMessage="Please wait..."></prx-spinner>
     <router-outlet name="sidenav"></router-outlet>
-    <section class="content" *ngIf="podcasts">
-      <metrics-menu-bar *ngIf="!isPodcastLoading && !isEpisodeLoading"></metrics-menu-bar>
+    <section class="content" *ngIf="loaded$ | async">
```

Comment:
Uses loading$ and loaded$ fixes #74. (I was testing on a series without a feederId this morning, and that was causing other swallowed errors that I was not realizing.)